outport/process/alteredaccounts/tokensProcessor.go

```diff
@@ -111,24 +115,50 @@ func (tp *tokensProcessor) extractEsdtData(
 
 	if eventShouldContainReceiverAddress && len(topics) > idxReceiverAddressInTopics {
 		destinationAddress := topics[idxReceiverAddressInTopics]
-		err = tp.processEsdtDataForAddress(destinationAddress, nonce, string(tokenID), markedAlteredAccounts, false)
-		if err != nil {
-			return err
-		}
+		tp.processEsdtDataForAddress(destinationAddress, nonce, string(tokenID), markedAlteredAccounts, false)
 	}
 
 	return nil
 }
 
+func (tp *tokensProcessor) processMultiTransferEventV2(event data.EventHandler, markedAlteredAccounts map[string]*markedAlteredAccount) {
+	topics := event.GetTopics()
+	// MultiESDTNFTTransfer V2 event
+	// N = len(topics)
+	// i := 0; i < N-1; i+=3
+	// {
+	// 		topics[i] --- token identifier
+	// 		topics[i+1] --- token nonce
+	// 		topics[i+2] --- transferred value
+	// }
+	// topics[N-1]   --- destination address
+	if len(topics) < minTopicsMultiTransferV2 {
+		return
+	}
+
```

Comment:
might add here a check like:
```GO
if len(topics) % 3 != 1 {
     log.Warn("programming error.....
     return
}
```
the log.Warn can also be on L136

Reply:
done